tests/bdd/features/snapshot/clone/create.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Volume Snapshot Clone Creation
+
+  Background:
+    Given a deployer cluster
+    And a valid snapshot of a single replica volume
+
+  Scenario: Create a new volume as a snapshot clone from a valid snapshot
+    When we create a new volume with the snapshot as its source
+    Then a new replica will be created for the new volume
+    And the replica's capacity will be same as the snapshot
+
+  Scenario: Create multiple new volumes as snapshot clones for a valid snapshot
+    When we attempt to create 4 new volumes with the snapshot as their source
+    Then all requests should succeed
```

Comment:
This one is good.

Can we also add a test case where we attempt continuous snapping and cloning from the newly created clones.

For ex, 
Vol 1 -> Snap 1 -> Clone 1
Clone 1 -> Snap 2 -> Clone 2
Clone 2 -> Snap 3 -> Clone 3

Reply:
Done :)